node/core/backing/src/error.rs

```diff
@@ -70,6 +70,10 @@ pub enum Error {
 
 	#[error(transparent)]
 	SubsystemError(#[from] SubsystemError),
+
+	#[fatal]
+	#[error(transparent)]
+	OverseerExited(SubsystemError),
```

Comment:
Missing `#[from]` to keep the error source.

```suggestion
	OverseerExited(#[from] SubsystemError),
```

Reply:
This is already ensured by using transparent. From would also have been wrong, because we already have the from implementation for subsystem error for another variant.